addons/web/static/src/views/fields/image/image_field.js

```diff
@@ -43,9 +44,34 @@ export class ImageField extends Component {
             isValid: true,
         });
         this.lastURL = undefined;
+
+        if (this.props.record.fields[this.props.name].related) {
+            this.lastUpdate = this._now();
+            const cacheKey = () => ({
+                value: this.props.value,
+                resId: this.props.record.resId,
```

Comment:
Is `resId` useful in the key?
When the record is created, the value changes at the same time (from base64 to a size notation), isn't it?